### ISSUE TITLE
Completely broken in Vanilla 2.3

### ISSUE THREAD

**Issue by AeSix** [CodeSequence/sanmyaku#49]
There are large "X" which are not constrained by ctrl+/ctrl- with no images. 
I have taken a screenshot, however I had to make the browser much more narrow than I usually use it in order to shrink the Xs small enough to take the image. 

Your theme is really beautiful and I'd really love to use it, but obviously cannot do so with Vanilla 2.3, or presumably 2.4b. 
![06cd03c2da](https://cloud.githubusercontent.com/assets/5191757/24418622/8e99f8fa-13ba-11e7-93c0-1090850a6d07.png)

**Comment by valeryan** [comment]
@AeSix are you using the pre built release or did you pull the theme from github? If you pulled it from github did you go through the build steps. I just downloaded a copy of vanilla 2.3, pulled our theme fresh, and built it. The theme works perfectly with vanilla 2.3 for me. 

These are the basic steps I used (note: assuming you have node, npm, gulp, and maybe yarn installed):
`cd ~/vanilla/themes/`
`git clone https://github.com/CodeSequence/sanmyaku.git`
`cd sanmyaku`
`npm install` or `yarn`
`gulp build`

Now go set sanmayku as your default theme and view vanilla.
